JiraPS/Public/New-JiraIssue.ps1

```diff
@@ -2,59 +2,67 @@ function New-JiraIssue {
     # .ExternalHelp ..\JiraPS-help.xml
     [CmdletBinding( SupportsShouldProcess )]
     param(
-        [Parameter( Mandatory )]
+        [Parameter( Mandatory, ValueFromPipelineByPropertyName )]
         [String]
         $Project,
 
-        [Parameter( Mandatory )]
+        [Parameter( Mandatory, ValueFromPipelineByPropertyName )]
         [String]
         $IssueType,
 
-        [Parameter( Mandatory )]
+        [Parameter( Mandatory, ValueFromPipelineByPropertyName )]
         [String]
         $Summary,
 
+        [Parameter( ValueFromPipelineByPropertyName )]
         [Int]
         $Priority,
 
+        [Parameter( ValueFromPipelineByPropertyName )]
         [String]
         $Description,
 
+        [Parameter( ValueFromPipelineByPropertyName )]
         [AllowNull()]
         [AllowEmptyString()]
         [String]
         $Reporter,
 
+        [Parameter( ValueFromPipelineByPropertyName )]
         [String[]]
         $Labels,
 
+        [Parameter( ValueFromPipelineByPropertyName )]
         [String]
         $Parent,
 
+        [Parameter( ValueFromPipelineByPropertyName )]
         [Alias('FixVersions')]
         [String[]]
         $FixVersion,
 
+        [Parameter( ValueFromPipelineByPropertyName )]
         [PSCustomObject]
         $Fields,
 
-        [Parameter()]
+        [Parameter( ValueFromPipelineByPropertyName )]
```

Comment:
`Credential` is not part of any object that can be used in the pipeline.
please remove this change